imports/plugins/included/static-pages/client/templates/static-pages.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-undef */
+import { Template } from "meteor/templating";
+import "./static-pages.html";
+import { StaticPages } from "/lib/collections";
+import { Reaction } from "/client/api";
+
+Template.staticPages.onCreated(() => {
+   // Subscribe to Pages publication
+  Meteor.subscribe("staticPages");
+});
```

Comment:
I think it is better to use subscription Ready method immediately after you subscribe to a particular publication, due to that fact that the result might be needed by your code when it is not ready ---( Javascript synchronous )